src/Worker/Worker.php

```diff
@@ -65,4 +86,30 @@ public function process(MessageInterface $message, Queue $queue): void
             }
         }
     }
+
+    private function getHandler(string $name): ?callable
+    {
+        if (isset($this->handlersCached[$name])) {
+            return $this->handlersCached[$name];
+        }
+
+        $this->handlersCached[$name] = null;
+
+        $handler = $this->handlers[$name] ?? null;
+
+        if (is_callable($handler)) {
+            $this->handlersCached[$name] = $handler;
+        }
+
+        if (
+            is_array($handler)
```

Comment:
put it in the method, otherwise it is not easy to read

```php
    private function isHandlerFromContainer($handler): bool
    {
        return
            is_array($handler) &&
            array_keys($handler) === [0, 1] &&
            is_string($handler[0]);
    }
```